schema/annotation-source.yaml

```diff
@@ -36,14 +36,14 @@ $defs:
         description: >-
           A quantitative score reflecting the degree of confidence that the information 
           the information entity represents is true.
-      method:
+      specified_by:
         oneOf:
         - $ref: "#/$defs/Method"
         - $ref_curie: gks.core:CURIE
         description: >-
           A :ref:`Method` that describes all or part of the process through which the information was
           generated.
-      contributions:
+      contribution:
```

Comment:
This is confusing to me. As an array, there are multiple entities here, and `contributions` seems appropriate (similar to `members` in some VRS entities). @mbrush what is the rationale for this change request?

Reply:
Just how we did things in the RDF/LOD and ontology world - where I was taught to avoid use of plurals in naming properties/attributes.  But I suspect that the convention outside of this context is to use the plural form for attributes taking arrays as values. Totally fine to go with `contributions` here if that is the case.